backend/src/config/routes.rs

```diff
@@ -31,11 +31,18 @@ pub fn config(cfg: &mut web::ServiceConfig) {
                 .service(map::find_by_id)
                 .service(map::create)
                 .service(
-                    web::scope("/{map_id}/layers/plants/plantings")
-                        .service(plantings::find)
-                        .service(plantings::create)
-                        .service(plantings::update)
-                        .service(plantings::delete),
+                    web::scope("/{map_id}/layers")
+                        .service(layers::find)
+                        .service(layers::find_by_id)
+                        .service(layers::create)
+                        .service(layers::delete)
+                        .service(
+                            web::scope("plants/plantings")
```

Comment:
```suggestion
                            web::scope("plants")
```

Do we need the differentiation between plants layer and plantings on the plants layer here? Unless we want to have other, non-plantings related operations on the plants layer we can assume from context that we deal with plantings.

Reply:
We might at some point have special operations on the plants layer itself. So `/layers` are operations on all types of layer, `/layers/plants` would be operations on the plants layer (e.g. heatmap might fit better there than under plantings), and `/layers/plants/plantings` are objects on the plants layer.

Reply:
> We might at some point have special operations on the plants layer itself. So /layers are operations on all types of layer, /layers/plants would be operations on the plants layer

Yes.

> e.g. heatmap might fit better there than under plantings

Maybe but the heatmap is specific to individual plants, like the relations. Where do you suggest to put the heatmap?

I think we should put relations and heatmap together in an API call (as also suggested by runtime), mostly for performance reasons: to keep the number of API calls as low as possible. I wrote this now in #459 0c2c88cc62275d162c39865e579484e4bf591bf8

> and /layers/plants/plantings are objects on the plants layer.

Exactly.